### PR TITLE
Feature/add youtube story

### PR DESCRIPTION
**Pull request by spences10**

### CODE REVIEW

File: packages/mdx-embed/src/components/MDXEmbedProvider/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { MDXEmbedProvider } from './MDXEmbedProvider';
+export { MDXEmbedProvider } from './MdxEmbedProvider';
```
Comment (PaulieScanlon) [inline packages/mdx-embed/src/components/MDXEmbedProvider/index.ts:1]: Hmmm, this is gonna be a problem. I'd like to keep it as `MDXProvider`... can you revert this change? or does that mean you get errors?

**Review by PaulieScanlon — CHANGES_REQUESTED** [review]

File: packages/mdx-embed/src/components/MDXEmbedProvider/index.ts (hunk 1 above)
Comment (spences10) [reply packages/mdx-embed/src/components/MDXEmbedProvider/index.ts:1]: Yeah, I get errors when it's the original casing

Comment (spences10) [reply packages/mdx-embed/src/components/MDXEmbedProvider/index.ts:1]: This should be resolved with #15

**Comment by spences10** [comment]
I'll base this off of #13 

Could you put #15 first though @PaulieScanlon?
